platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ExportMigrationEntryImpl.java

```diff
@@ -172,6 +173,8 @@ public boolean store(boolean required) {
             if (absolutePathError instanceof NoSuchFileException) {
                 this.stored = storeWhenNoSuchFile(required);
             } else if (absolutePathError != null) {
+                SecurityLogger.audit("Exporting file {}", absolutePath);
+                SecurityLogger.audit("Error exporting file {}", absolutePath);
```

Comment:
These two audits right next to each other do not look right to me.

Reply:
The only reason I did it is because otherwise you would not have had the attempt to export the file audited since I generate the error right away. In case where we actually attempt to read it, we first audit the export/read and then the failure if one occurs.

Reply:
Made changed based on our discussion